v2/kafka-to-pubsub/src/main/resources/kafka_to_pubsub_metadata.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "Kafka to Pub/Sub",
+  "description": "Streaming pipeline. Ingests JSON-encoded messages from Kafka to a Pub/Sub topic",
+  "parameters": [
+    {
+      "name": "bootstrapServers",
+      "label": "Kafka Bootstrap Servers",
+      "helpText": "Comma separated kafka bootstrap servers in format ip:port",
+      "regexes": [],
+      "paramType": "TEXT"
+    },
+    {
+      "name": "inputTopics",
+      "label": "Kafka input topics",
+      "helpText": "Comma separated Kafka topics to read data",
+      "regexes": [],
+      "paramType": "TEXT"
+    },
+    {
+      "name": "outputTopic",
+      "label": "Pub/Sub output topic",
+      "helpText": "Pub/Sub topic to write the output, in the format of 'projects/yourproject/topics/yourtopic'",
+      "regexes": [
+        "^projects\\/[^\\n\\r\\/]+\\/topics\\/[^\\n\\r\\/]+$"
+      ],
+      "paramType": "TEXT"
+    },
+    {
```

Comment:
remove this as this is not used in pipeline

Reply:
Removed outputFormat parameter.